### DISCUSSION TITLE
Cloud Available?

### DISCUSSION THREAD

**Discussion by osseonews** [saleor/saleor#7579] [General]
Saleor seems like one of the best headless solutions currently, and we've been tracking its development closely. However, we have tried to sign up for the cloud service multiple times over the past 6 months and can never get set up. We just get an automated email response: "We’ve received your submission. We will try to respond as soon as possible." After this email, we never get a response. When do you expect that the cloud solution will actually be available? I assumed the cloud version is how Saleor makes $, so I am surprised it's not possible to set it up. Were you waiting for a stable 3.0 launch?

Thanks.

**Comment by karolkielecki** [comment]
Hi there! Thank you for your kind words regarding our headless platform. Saleor Cloud is still under development. It will be released in tandem with Saleor 3.0, so you will be able to self signup to try the sandbox. The beta version of [the 3.0 release](https://github.com/mirumee/saleor/releases/tag/3.0.0-b.3) is now available. There is no release day, but probably it will be around July/August. 

@osseonews
